docs/src/man/man1/hy_gt_vfd.1.adoc

```diff
@@ -257,9 +257,11 @@ PC.03 = 1::
 
         The VFD's external counter register.
 
-*hy_gt_vfd.fault-info* (float, out)::
+*hy_gt_vfd.fault-info* (u32, out)::
 
-        The VFD's fault info register.
+        The VFD's fault info register.  The type of this register was
+        changed from float to u32 in version 2.10.  The commit was
+        done 2022-12-17.
```

Comment:
This kind of user-facing changelog information belongs in [Updating LinuxCNC](http://linuxcnc.org/docs/devel/html/getting-started/updating-linuxcnc.html), not in the manpage.

Reply:
The patch was reworked to not break backward compatibility, so no note is needed in `Updating LinuxCNC`.